extensions/amp-form/0.1/test/test-amp-form-textarea.js

```diff
@@ -37,9 +38,18 @@ describes.realWin('amp-form textarea[autoexpand]', {
     sandbox = env.sandbox;
   });
 
-  describe('AmpFormTextarea', () => {
+  describe('handleInitialOverflowElements', () => {
     it('should remove autoexpand on elements with initial overflow', () => {
+      const textarea = doc.createElement('textarea');
+      textarea.setAttribute('autoexpand', '');
+      textarea.setAttribute('rows', '1');
+      textarea.setAttribute('cols', '80');
+      textarea.innerHTML = 'big text'.repeat(30);
+      doc.body.appendChild(textarea);
 
+      return handleInitialOverflowElements([textarea]).then(() => {
```

Comment:
async/await?

Reply:
That would probably improve the test readability overall, but since amp-form tests don't use async await yet, I prefer not to add it in this PR